### DISCUSSION TITLE
Playist always start a new stream after each video ?

### DISCUSSION THREAD

**Discussion by Rabbitpaaw-de** [ant-media/Ant-Media-Server#5364] [Q&A]
Hello dear community, I have the following problem and indeed I want to make a 24/7 stream on Twitch but when a video is finished and the next plays it always restarts the stream ? Is there a way that the stream remains active?

**Comment by yashtandon113** [comment]
HI @Rabbitpaaw-de 
Thanks for reporting the issue. There is one known issue created on Git Hub repository.
https://github.com/ant-media/Ant-Media-Server/issues/5005

As far as I remember, this issue only happens on Twitch but on YouTube there is no such issue. I will try to reproduce it and update you asap.

**Reply by Rabbitpaaw-de** [reply]
Thanks I like the tool and would also buy it if it works before with twitch :)

**Reply by mekya** [reply]
Hi @Rabbitpaaw-de ,
Could you please reach to solution team through contact@antmedia.io and say how many licenses you are going to get when this feature is available? So some priority can be set for this issue.  

Cheers

**Reply by mekya** [reply]
Hi @Rabbitpaaw-de,

I'm not sure if you reach the team or not. I just want to update you. We've provided a workaround solution for this problem. 

Check this out -> https://github.com/orgs/ant-media/discussions/6158
